### PR TITLE
Use "foo not in" instead of "not foo in"

### PR DESCRIPTION
**Pull request by JeroenDeDauw**

### CODE REVIEW

File: src/games/gamesContainer.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def findGameByUuid(self, uuid):
 
     def addGame(self, game):
         """Add a game to the list"""
-        if not game in self.games :
+        if game not in self.games :
```
Comment (Sheeo) [inline src/games/gamesContainer.py]: For this, please remove the prefix space before ":" as well :)

Comment (JeroenDeDauw) [inline src/games/gamesContainer.py]: At that rate we'll still have them by the time protons start decaying. What's the point of fixing it here if there are another 100 of these in the file? Better to fix them all in one go IMO.

Comment (Sheeo) [inline src/games/gamesContainer.py]: Just fixing them as we see them will reduce them slowly but surely, if you want to do a full scale commit for it that is fine too.

**Comment by JeroenDeDauw** [comment]
I've added a commit that fixes those things for this file. And some other stuff. Apparently PyCharm has no option to just fix the spacing stuff?

**Comment by Sheeo** [comment]
Will do this as we come by them.

**Pull request closed without merging**